packages/tools/src/tools/StackScrollTool.ts

```diff
@@ -22,32 +31,26 @@ class StackScrollTool extends BaseTool {
         invert: false,
         debounceIfNotLoaded: true,
         loop: false,
+        // this tool can also be used to rotate the volume for instance for MIP
```

Comment:
Can you bind rotate to the drag callback too?  It looks like that doesn't work, and drag/mouse move only scrolls.